projects/Mallard/android/app/src/main/java/com/guardian/editions/MainApplication.java

```diff
@@ -68,5 +69,32 @@ public ReactNativeHost getReactNativeHost() {
     public void onCreate() {
         super.onCreate();
         SoLoader.init(this, /* native exopackage */ false);
+        initializeFlipper(this); // Remove this line if you don't want Flipper enabled
+    }
+
+  /**
+   * Loads Flipper in React Native templates.
+   *
+   * @param context
+   */
+  private static void initializeFlipper(Context context) {
```

Comment:
What is this? I missed this when doing the upgrade.

Reply:
What's flipper?

Reply:
Reflection - urgh.

Reply:
https://github.com/facebook/flipper